### PR TITLE
Change-default-branch-name-to-main

### PR DESCRIPTION
**Pull request by cprivitere**
This PR would only be merged once the default branch name was switched to main.

Signed-off-by: Chris Privitere <23177737+cprivitere@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 `cloud-provider-equinix-metal` is the Kubernetes CCM implementation for Equinix Metal. Read more about the CCM in [the official Kubernetes documentation](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/).
 
-This repository is [Maintained](https://github.com/packethost/standards/blob/master/maintained-statement.md)!
+This repository is [Maintained](https://github.com/packethost/standards/blob/main/maintained-statement.md)!
```
Comment (displague) [inline README.md:16]: nit: this could be more rewritten to skip a redirect https://github.com/equinix-labs/equinix-labs/blob/main/maintained-statement.md

Comment (displague) [reply README.md:16]: Same with the other packethost/standards repo references

File: deploy/chart/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: cloud-provider-equinix-metal
 description: Kubernetes Cloud Provider for Equinix Metal 
 type: application
 version: "0.1.0"
-appVersion: "master"
+appVersion: "main"
```
Comment (displague) [inline deploy/chart/Chart.yaml:6]: Are there `helm upgrade` implications to this change?

**Review by displague — APPROVED** [review]
I have one question about the `helm upgrade` path, but I don't believe Helm is a popular install mechanism for this project. We may need to call the appVersion change out in the release notes if it isn't seamlessly handled.

**Review by ctreatma — APPROVED** [review]

**Comment by ctreatma** [comment]
Based on the [Chart.yaml docs for that field](https://helm.sh/docs/topics/charts/#the-appversion-field), I think the `appVersion` will not impact users:

> This field is informational, and has no impact on chart version calculations.

File: deploy/chart/Chart.yaml (hunk 2 above)
Comment (cprivitere) [reply deploy/chart/Chart.yaml:6]: I don't believe so as it's my understanding that this helm doesn't actually exist anywhere other than this chart file. It's just an experiment.

File: README.md (hunk 1 above)
Comment (cprivitere) [reply README.md]: I've replaced the two instances I could find.

**Review by ctreatma — APPROVED** [review]

**Pull request merged**